cookbook-connector/src/test/resources/AutomationSpringBeans.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Do we required this beans file ? I dont thing you have a legacy functional test here. Please remove this if not required.